AN00209_xCORE-200_DSP_Library/app_adaptive/src/app_adaptive.xc

```diff
@@ -69,38 +84,48 @@ int32_t fir_state[FIR_FILTER_LENGTH];
 int32_t lms_coeffs[FIR_FILTER_LENGTH];
 int32_t nlms_coeffs[FIR_FILTER_LENGTH];
 
-int main(void)
-{
+int main(void) {
   int32_t c;
   int32_t x;
   int32_t err;
 
-              // Apply LMS Filter
-  for( c = 5; c <= 5; c *= 2 )
-  {
-    printf( "LMS %u\n", c );
-    for( int32_t i = 0; i < FIR_FILTER_LENGTH; ++i ) lms_coeffs[i] = fir_coeffs[i];
-    for( int32_t i = 0; i < FIR_FILTER_LENGTH; ++i ) fir_state[i] = 0;
-    for( int32_t i = 0; i < c+30; ++i )
-    {
-      x = dsp_adaptive_lms( Q31(0.08), Q31(0.10), &err, lms_coeffs, fir_state, c, Q31(0.01), Q_N );
-      print31( x ); print31( err ); printf( "\n" );
+  // Apply LMS Filter
+  for (c = 5; c <= 5; c *= 2) {
+    printf("LMS %u\n", c);
+    for (int32_t i = 0; i < FIR_FILTER_LENGTH; ++i)
+      lms_coeffs[i] = fir_coeffs[i];
+    for (int32_t i = 0; i < FIR_FILTER_LENGTH; ++i)
+      fir_state[i] = 0;
+    for (int32_t i = 0; i < c + 30; ++i) {
+      x = dsp_adaptive_lms(
+          Q31(0.08), Q31(0.10), &err, lms_coeffs, fir_state, c, Q31(0.01), Q_N);
+      print31(x);
+      print31(err);
+      printf("\n");
     }
   }
 
-              // Apply Normalized LMS Filter
-  for( c = 5; c <= 5; c *= 2 )
-  {
-    printf( "\nNormalized LMS %u\n", c );
-    for( int32_t i = 0; i < FIR_FILTER_LENGTH; ++i ) nlms_coeffs[i] = fir_coeffs[i];
-    for( int32_t i = 0; i < FIR_FILTER_LENGTH; ++i ) fir_state[i] = 0;
-    for( int32_t i = 0; i < c+30; ++i )
-    {
-      x = dsp_adaptive_nlms( Q31(0.08), Q31(0.10), &err, nlms_coeffs, fir_state, c, Q31(0.01), Q_N );
-      print31( x ); print31( err ); printf( "\n" );
+  // Apply Normalized LMS Filter
+  for (c = 5; c <= 5; c *= 2) {
+    printf("\nNormalized LMS %u\n", c);
+    for (int32_t i = 0; i < FIR_FILTER_LENGTH; ++i)
+      nlms_coeffs[i] = fir_coeffs[i];
+    for (int32_t i = 0; i < FIR_FILTER_LENGTH; ++i)
+      fir_state[i] = 0;
+    for (int32_t i = 0; i < c + 30; ++i) {
+      x = dsp_adaptive_nlms(Q31(0.08),
```

Comment:
I think we'll want to look at the options to control how it breaks up parameter lists like this.